src/Myra/Graphics2D/UI/TextBox.cs

```diff
@@ -368,6 +368,18 @@ public TextBox(string styleName = Stylesheet.DefaultStyleName)
 			SetStyle(styleName);
 
 			BlinkIntervalInMs = 450;
+
+			KeyboardFocusChanged += (e, a) =>
```

Comment:
Is this code needed? As far as I remember, KeyboardFocusChanged is being fired by when property Widget.IsKeyboardFocused is being changed(which - in its turn - happens when Widget.OnLostKeyboardFocus/OnGotKeyboardFocus are called)

Reply:
From what I can tell, the event wasn't firing for the textbox at least. I set my break point and changed the focus and at no point was it called.

It could probably be moved to an actual function rather than a lambda though, if you prefer.

Reply:
I see, thanks for the explanation. Will take a look at it.